versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -24,6 +24,10 @@ project, you must also update the
 composer require front-commerce/magento2-commerce-module@2.1.0
 ```
 
+### New features in `2.23.0`
+
+- [Use Magento's `storeConfig` GraphQL query](/docs/2.x/magento2/using-magento-configuration#use-magentos-storeconfig-graphql-query) to access configurations with the `MagentoGraphQLConfig` loader
+
```

Comment:
**question:** should features be listed in the migration guide ?

Reply:
Yes, for the ones we think that integrators could use to refactor and improve their codebase.